src/lib.rs

```diff
@@ -38,6 +38,13 @@
 //! mutate a `Model`. It might not be completely stable at the moment so use
 //! with care (or not at all!).
 //!
+//! ## `gtfs`
+//! This feature is only used to expose some gtfs functions for use in external projects
```

Comment:
I would mention that the exposed API are not stable and that using this is experimental. I'm not expecting us to be as careful about the stability here that for the rest of the exposed API. What do you think?

Reply:
done in 6f11bef5115732b36972ed4fe3f9e8477c629b99